data/2024/2024-08-06/readme.md

```diff
@@ -0,0 +1,71 @@
+# Olympics athletes and medals
+
+This week we're exploring Olympics data! 
+
+The data this week comes from the [RGriffin Kaggle dataset: 120 years of Olympic history: athletes and results](https://www.kaggle.com/datasets/heesoo37/120-years-of-olympic-history-athletes-and-results/), basic bio data on athletes and medal results from Athens 1896 to Rio 2016. 
+
+This is an intentional repeat of [TidyTuesday 2021-07-27](https://github.com/rfordatascience/tidytuesday/tree/master/data/2021/2021-07-27), since it's the Olympics!
```

Comment:
```suggestion
This is an intentional repeat of [TidyTuesday 2021-07-27](https://tidytues.day/2021/2021-07-27), since it's the Olympics!
```
Use the tidytues.day url to better future-proof against changes in the structure of the repo.